docs/guidance/map/framework_agreements.md

```diff
@@ -36,13 +36,13 @@ Framework agreements can be **open** or **closed**:
 
 Procurement systems which provide for closed framework agreements include the EU (Framework Agreements, per Directive 2014/24/EU, Article 33) and the US federal system (Indefinite-delivery contracts, per Federal Acquisition Regulation, Subpart 16.5). An example of a procurement system which provides for open framework agreements is the EU (Dynamic Purchasing Systems, per Directive 2014/24/EU, Article 33).
 
-Framework agreement procedures can involve second-stage competition, or not:
+Framework agreement procedures can involve second stage competition, or not:
```

Comment:
Please restore the hyphen. It's how UNCITRAL styles it, and it is easier to parse.

Reply:
Restored.